### ISSUE TITLE
[BUG] Update to 2.0.8 and Agora no longer works on mobile devices - CRITICAL

### ISSUE THREAD

**Issue by geotraveler** [AgoraIO/Agora-WordPress#98]
**Describe the bug**

This is a critical issue. After updating to the latest version, Agora not longer works from a mobile device. All the buttons are not clickable and the video/audio doesn't work. It works well when viewing from a laptop, but can no longer connect from a mobile device. I tried rolling back . The dropdown for the audio and video devices no longer work as well.

This is a critical issue as we're going live tomorrow.

**To Reproduce**
1. Please contact us for an internal link to test

**Comment by digitallysavvy** [comment]
@geotraveler following up on our email conversation, you were able to use the plugin from mobile, are you still experiencing this issue on any devices?

**Comment by geotraveler** [comment]
Hej! Still having the issues and since we needed to go live, we used Zoom as a backup solution for this particular client.
Agora is still our preference but it needs to work. It feels so unstable, especially the audio issues. And we don't know what to do at this point. Could it be issues related to the latest WordPress upgrade? I know that upgrade broke a lot of plugins so we're using the temp fix Enable jQuery plugin to step down to a legacy wp jQuery library.
